pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -661,7 +661,12 @@ func (cm *containerManagerImpl) GetResources(pod *v1.Pod, container *v1.Containe
 		if err != nil {
 			return nil, err
 		}
+		// NOTE: Passing CDI device names as annotations is a temporary solution
```

Comment:
So for now we send it both as annotations and in the new `CDIDevices` as well right?

Reply:
yes, we do it for compatibility reasons as runtimes get CDI device names from annotations. After this PR is merged we're going to switch runtimes to use new CRI field instead of annotations. After that we'll remove annotation support from the Kubelet.